### PR TITLE
First Microverse Project

### PR DESCRIPTION
**Pull request by 1DelaMcRaph**
merged the first PR...resubmiting for review

### CODE REVIEW

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @1DelaMcRaph , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Good commit messages :heavy_check_mark: 
- Good use of the `.gitignore` file :heavy_check_mark: 
- All relevant files created :heavy_check_mark: 

## Required Changes ♻️
- [x]  Great job so far @1DelaMcRaph However, as per the project requirements, you should ensure that you used the correct ([GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)) in your project. You can use the link above to read more about how this can be done. I have also noted it in the image below :100: 
- [x] Also, please ensure that your branch contains a short descriptive summary of the changes that were made in the current pull request :100: 
- [x] As mentioned in the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), you are to set up [linters for HTML and CSS](https://github.com/microverseinc/linters-config) but currently, you have no linters set up in your project so kindly do this as well :100: 
- [x] I also noticed that you are missing the  [Microverse Readme template](https://github.com/microverseinc/readme-template). This makes your project incomplete. Every section of the readme template is required with the exception of the FAQ section which is optional. So please make sure to add the README template to your project and fill in all the necessary contents :+1:

- [x] Follow the steps I have listed below to customize it to suit this project:

    • Remove the Microverse logo. If you have your logo, you can use it but I would advise you to remove it :100: 

    • For the Tech Stack Section, Kindly remove the Server and Database subsections because the technologies listed there are not relevant to this project. For the Client section, kindly include HTML and CSS :+1: 

    • For the Key Features section, kindly highlight what this project is all about. This will make it easy for those who go through this repository to know what was implemented by reading the README.md file. :wink: 

    • For the Prerequisite Section, kindly add tools that are needed before you can clone this repository on a local machine. Tools like:
        ◦ Installation of Git
        ◦ Installation of NPM
        ◦ A working browser
        ◦ Internet connection
        ◦ A code editor like Visual Studio Code
        ◦ Installation of Live Server plugging in Visual Studio Code
        ◦ Any other tool you can think about :100: 

    • For the Setup section, kindly add instructions for cloning your repository on a local machine :+1: 

    • For the Install section, kindly add instructions for setting up linters used in this project :+1: 

    • For the Usage section, include instructions for opening this project on a browser. For example, opening the project by launching the Live Server plugging in visual studio code or any other means that you know.

- For the Future Features section include 2 or more future features.

- Please include an MIT license file in your project. The link to your license should be included in your README file. The MIT License is a popular open-source license that allows others to use, modify, and distribute your code as long as they give credit to you the original author, and include the original license text. You can use this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to learn how to add one to your project :wink: 

 - Kindly implement all the changes listed above. It will make your README.md file look professional and it will give you an idea of how to set README.md files for most of the projects you will execute as a Microverse student.  
- [x] You might need to create a new pull request in order to follow the correct GitHub flow. So I will suggest you reference this pull request in the new Pull request you'll create so the next reviewer will have access to my comments and be able to approve your project if everything was completed when you submit your project for a rereview  :100: 

image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbelG101 — COMMENTED** [review]
# Hi @1DelaMcRaph  👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Since the GitHub workflow is not correct on this PR and you've submitted another one you just have to provide a link to this PR on your 2nd PR and not resubmit this again. ✔

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: .gitignore [hunk 1]
```diff
@@ -128,6 +128,3 @@ dist
 .yarn/build-state.yml
 .yarn/install-state.gz
```
Comment (bobb-Rob) [inline .gitignore]: - [ ] In your README file, some sections are missing, they include 👇🏾 
  -  Authors
  - Contributing
  - Show your support
  - Acknowledgement
- [ ] Also, you linter checks are failing. Check the details for each failing check and see what you need to do.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️

Hi @1DelaMcRaph, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**